tests/acceptance/crud-failure-test.js

```diff
@@ -167,6 +177,41 @@ test('Create field errors', function(assert) {
   });
 });
 
+test('Created nested field errors', function(assert) {
+  assert.expect(8);
+
+  return Ember.run(function() {
+    var post = store.createRecord('post');
+    var embeddedPostComment = store.createRecord('embedded-post-comment', {
+      body: 'This is my new comment',
+      post: post
+    });
+
+    return embeddedPostComment.save().then(null, function(response) {
+      const comment = embeddedPostComment;
+      assert.ok(response);
+      assert.ok(response.errors);
+      assert.notOk(comment.get('isValid'));
+
+      /*
+      JSON API technically does not allow nesting, so the nested errors are
+      processed as if they were attributes on the comment itself.  As a result,
+      comment.get('post.isValid') returns true :-(
+      */
```

Comment:
It would be nice to have a link to the json-api discussion in the comment (json-api/json-api#899) - just so it's easy to find in the future.